knowledge_repo/app/static/css/pages/base.css

```diff
@@ -801,3 +905,27 @@ border-color: #11b6aa;
   height:70px; /* fixed header height*/
   margin:-70px 0 0; /* negative fixed header height */
 }
+
+h5 {
+  margin-block-start: 11px;
+  margin-block-end: 0px;
+}
+
+
+
+.select2-container .select2-selection--multiple {
+    background-color: white;
+    border: 1px solid #e2ddddcf!important;
```

Comment:
generally using `!important` is not recommended. Is there another way you can set these attributes?

Reply:
I actually tried set it multiple ways and couldn't get it work, so I ended up using !important.

Reply:
what exactly are you trying to accomplish here? Maybe you can add a new css class to the element you're trying to style.

Reply:
These are styling the interactive author and tags entry fields that have autosuggest functionality. The styles are to have them match with the rest of the UI. These come from the `select2` library and I found I couldn't get the style customization to apply unless I marked them as important.

Reply:
Finally got this to work. Referred to the class by a parent ID